### ISSUE TITLE
Tag a release and image transparency?

### ISSUE THREAD

**Issue by bjlittle** [ed-hawkins/show-your-stripes#1]
Hi @ed-hawkins!

Congratulations and thank you so much to you and your team for the amazing [#ShowYourStripes](https://showyourstripes.info/s/globe/) campaign :heart_eyes: 

We're so super proud to fully support your work over on our [SciTools/iris](https://github.com/SciTools/iris) repository, which is primarily maintained by core developers at the UK Met Office, by adding a reference to your work in our `README.md` (PR https://github.com/SciTools/iris/pull/5141).

I do have two small requests for you to consider though, which would help us and possibly others in the community:

1. Would it be possible to **tag a release** of your repository whenever you update it with new yearly images? That way we can refer to the de-facto release `tag` rather than your `master` branch.
2. Would it be possible to generate your `png` climate stripe images with a **transparent background** instead of a black background? As this would allow the images to have the same colour as the host publication site.

The metadata in your `png`s suggests that you're using `matlab` to generate the images? I'm not a `matlab` user myself, but it **seems** possible to achieve transparency with `matlab` given this [stackoverflow Q&A thread](https://stackoverflow.com/questions/16134642/how-to-save-figure-with-transparent-background) discussion.

Many thanks again :+1:

**Comment by ed-hawkins** [comment]
Thanks Bill - yes to (1), and will look at the possibility for (2)! Ed.

**Comment by bjlittle** [comment]
That's really awesome, thanks @ed-hawkins :+1: 

Since I've got your attention :wink: ... is it easy to also generate `svg` as well as `png` images?

**Comment by ed-hawkins** [comment]
Yes, svg is possible for specific images. Which ones would you want? Ed.

**Comment by bjlittle** [comment]
The reason that I'm asking is that we're using the `GLOBE---1850-2021-MO.png` image, and resizing it only using HTML (not CSS) to render it as a thinner band i.e., see [here](https://github.com/SciTools/iris#showyourstripes).

However, we noticed that image blurring can occur with some browsers e.g., `chrome` and `edge`, but this wouldn't be the case with a `svg` rather than a `png` image.

That said, your [2021](https://github.com/ed-hawkins/show-your-stripes/tree/master/2021) collection contains 2000 images, but we'd only be initially interested in the 4 `GLOBE` images atm.

Obviously, others in the community may be interested in specific regions, so offering both `png` and `svg` would be the ultimate win, I guess. 

In the meantime, only `GLOBE` would be really helpful to us, if possible?

Many thanks!

**Comment by ed-hawkins** [comment]
Ok - can make svg versions of the global stripes in this year's release. Thanks.

**Comment by bjlittle** [comment]
@ed-hawkins Oh man... that's just fantastic news!

Thanks so much :beers:

**Comment by bjlittle** [comment]
@ed-hawkins BTW I've now also cited #showyourstripes on [geovista](https://github.com/bjlittle/geovista#license) :+1:

**Comment by bjlittle** [comment]
> Ok - can make svg versions of the global stripes in this year's release. Thanks.

Hi @ed-hawkins,

Just curious, but have you scheduled a release date for this year sometime?

Many thanks :+1:
